.github/workflows/zephyr.yml

```diff
@@ -11,15 +11,23 @@ jobs:
   zephyr-build:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         # From time to time this will catch a git tag and change SOF_VERSION
         with: {fetch-depth: 10, submodules: recursive}
 
+      # v0.23.4 is the last image with Zephyr SDK 0.14.
+      # SDK 0.15 fails with the following Werror:
+      #   /workdir/zephyrproject/zephyr/include/zephyr/kernel/thread_stack.h:190:16:
+      #   error: ignoring attribute 'section (".cached.\"WEST_TOPDIR/zephyr/kernel/init.c\".3")'
+      #   because it conflicts with previous 'section
+      #   (".cached.\"WEST_TOPDIR/zephyr/arch/xtensa/include/kernel_arch_func.h\"'
+      #   [-Werror=attributes]
+
       # https://github.com/zephyrproject-rtos/docker-image
       # Note: env variables can be passed to the container with
       # -e https_proxy=...
       - name: build
         run: docker run -v "$(pwd)":/workdir
-             ghcr.io/zephyrproject-rtos/zephyr-build:latest
+             ghcr.io/zephyrproject-rtos/zephyr-build:v0.23.4
```

Comment:
We are explicitly changing the behavior here?  Changing from the "moving target" of "latest" to a hard-coded version?  Does that mean someone has to make sure to keep this up-to-date?

Reply:
Yes because this is a stable branch, see commit message.